Bug: [ERROR] Unable to load config file: error="read /config/config.json: is a directory"

### Terraform Version
terraform v1.0.5

### Affected Resource(s)
docker-compose.yml

### Terraform Configuration Files / Docker compose file
```yml
version: '3.7'
services:
  api:
    image: "hashicorpdemoapp/product-api:v4280cf7"
    ports:
      - "19090:9090"
    volumes:
      - ${PWD}/conf.json:/config/config.json
    environment:
      CONFIG_FILE: '/config/config.json'
    depends_on:
      - db
  db:
    image: "hashicorpdemoapp/product-api-db:v4280cf7"
    ports:
      - "15432:5432"
    environment:
      POSTGRES_DB: 'products'
      POSTGRES_USER: 'postgres'
      POSTGRES_PASSWORD: 'password'
```

### Debug Output

```
api_1  | 2021-09-02T18:56:57.001Z [ERROR] Unable to load config file: error="read /config/config.json: is a directory"
```

### Panic Output
```
docker_compose_api_1 exited with code 1
```


### Expected Behavior
docker-compose up should be successfully spinning up the container. 

### Actual Behavior
docker compose exited with code 1, Hashicups api is not reachable. Cannot continue the [toturial](https://learn.hashicorp.com/tutorials/terraform/provider-setup?in=terraform/providers)

### Steps to Reproduce
Follow the guide listed in toturial ["Setup and implement Read"](https://learn.hashicorp.com/tutorials/terraform/provider-setup?in=terraform/providers)

### Suggested solution

Change from "${PWD}/conf.json:/config/config.json" to "./conf.json:/config/config.json" in line [L#8](https://github.com/hashicorp/terraform-provider-hashicups/blob/eb3b833280d3eaa911ea99c9539c0c4b7ad6de37/docker_compose/docker-compose.yml#L8)

--- a/docker_compose/docker-compose.yml
+++ b/docker_compose/docker-compose.yml
@@ -5,7 +5,7 @@ services:
     ports:
       - "19090:9090"
     volumes:
-      - ${PWD}/conf.json:/config/config.json
+      - ./conf.json:/config/config.json
     environment:
       CONFIG_FILE: '/config/config.json'
     depends_on:

